src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -775,6 +779,17 @@ public static void setup(SlimefunPlugin plugin) {
 		    .register(plugin);
 		}
 
+		if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_14)) {
```

Comment:
Why is this 1.14 only?

Reply:
because the convert method for zombie villagers were added in 1.14